src/large_pages/node_large_page.cc

```diff
@@ -24,6 +24,10 @@
 #include "util.h"
 #include "uv.h"
 
+#if defined(__DragonFly__)
+#error "Large pages is not supported by this platform"
+#endif
```

Comment:
It would be nicer to error out in configure.py rather than defer until build time. It should preferably also be its own separate commit.

Reply:
That comes more from the fact dragonflybsd "is" FreeBSD from configure standpoint.